### ISSUE TITLE
More info about errors/warnings in Refine

### ISSUE THREAD

**Issue by kristenwilson** [k-int/gokb-phase1#210]
Provide link to documentation in the Wiki about each message type. Need a form in CRED to manage documentation URLs.

**Comment by iyawnis** [comment]
@kristenwilson does the documentation for these errors already exist? If it does, where can I find it?

**Comment by kristenwilson** [comment]
No, it doesn't. Looking at this issue, I think this is no longer a priority. As we've done training, we've found that people became accustomed to the error/warnings very quickly and they're really pretty self explanatory. I'm going to bump this back to future and we can revisit if needed.
